Take 4: Leadership chapter with 3 new subchapters and landing page (#3136)
alan-turing-institute/the-turing-way: book/website/collaboration/leadership.md

There is no such thing as a fixed set of leadership skills and not all great leaders have the same traits, strengths or [leadership styles](https://www.skillsyouneed.com/lead/leadership-styles.html).

In the context of data science, where we strive to advance our knowledge through validated scientific methods based on data, talking about leadership, essentially a nuanced and fuzzy concept, is challenging to define.
In data science and research, discussions and upskilling in leadership are extremely important as:
* Researchers and data scientists constantly navigate fast-changing demands of 'innovative' approaches in technological development.
* They manage and coordinate collaborations with their peers as well as external stakeholders across multiple disciplines. 
* Leadership is one of the toughest non-technical skills to learn without guidance.
* Leadership is rarely discussed openly in research environments, despite being everywhere.
* (Open) Leadership is one of the keys to healthy and inclusive communities that we are striving to build in data science.

> In this chapter, we explore different aspects of (open leadership) that are applicable in the data science and research contexts. 

Author: ```suggestion
In this chapter, we explore different aspects of (open) leadership skills that are applicable in the data science and research contexts. 
```

After all, "technical skills are just one aspect of making data science research open, reproducible, inclusive and ethical for all" ([ref](https://the-turing-way.netlify.app/)).

We hope that by discussing leadership here, we will:
* Spotlight on important points that everyone in research should know
* Inspire those who think leadership is not for them to review their assumptions (and maybe see themselves with abilities to lead a project).
* Ignite reflection and spark conversations about the leaderships we want to promote in data science.
* Evaluate if leadership in our workplace or communities are healthy, compassionate, and inclusive, and how we can improve it.

In this chapter, we will introduce and discuss: 
- {ref}`important features<cl-leadership-features>` of leadership when experiencing and exercising them in data science,
- what we mean by {ref}`"healthy" leadership<cl-leadership-building>`, how we can build open and healthy leadership approaches and what they should avoid/address,
- how we can create {ref}`leadership opportunities <cl-leadership-opportunities>` in our projects,

> - finally, we invite you, the diverse group of leaders in *The Turing Way* (readers, users, contributors, and community members) to provide examples from your contexts.

Author: ```suggestion
- finally, our community members provide examples of open leadership from their own professions and experiences.
We invite you, the diverse group of leaders in *The Turing Way* (readers, users, contributors, and community members) to provide more examples from your contexts in this chapter.
```

In the context of data science, where we strive to advance our knowledge through validated scientific methods based on data, talking about leadership, essentially a nuanced and fuzzy concept, is challenging to define.
In data science and research, discussions and upskilling in leadership are extremely important as:
* Researchers and data scientists constantly navigate fast-changing demands of 'innovative' approaches in technological development.
* They manage and coordinate collaborations with their peers as well as external stakeholders across multiple disciplines. 
* Leadership is one of the toughest non-technical skills to learn without guidance.
* Leadership is rarely discussed openly in research environments, despite being everywhere.
* (Open) Leadership is one of the keys to healthy and inclusive communities that we are striving to build in data science.

In this chapter, we explore different aspects of (open) leadership skills that are applicable in the data science and research contexts. 

> After all, "technical skills are just one aspect of making data science research open, reproducible, inclusive and ethical for all" ([ref](https://the-turing-way.netlify.app/)).

Author: ```suggestion
After all, "technical skills are just one aspect of making data science research open, reproducible, inclusive and ethical for all" (as stated in the [Welcome page of The Turing Way](https://the-turing-way.netlify.app/)).
```
